src/fixturelibrary.ts

```diff
@@ -174,7 +174,7 @@ export class FixtureLibrary {
 
     const updatedFixtures: string[] = [];
 
-    ofl?.forEach(async (fixture) => {
+    await Promise.all((ofl ?? []).map(async (fixture) => {
```

Comment:
maybe do the `ofl ?? []` in the definition above? ends up being a bit cleaner:

```ts
    const ofl = await fetchOflFixtureDirectory() ?? [];

    const updatedFixtures: string[] = [];

    await Promise.all(ofl.map(async (fixture) => {
```

 (also same comment but for below)